bats/tests/containers/platform.bats

```diff
@@ -30,6 +30,7 @@ check_uname() {
 }
 
 @test 'deploy arm64 container' {
+    is_linux && skip
```

Comment:
Why is this necessary? This is supposed to work everywhere because we configure qemu-static emulation for the arm64 on the x86_64 VM (I think we are missing this in the WSL distro, but Linux uses the same ISO as macOS). Did this actually fail for you?

Reply:
Actually it works, I'll remove that line.